wurst/systems/spells/ElkCritter.wurst

```diff
@@ -0,0 +1,8 @@
+package ElkCritter
+import AbilityObjEditing
+import ObjectIdGenerator
+
+public let SPELL_ELK_CRITTER_ID = compiletime(ABIL_ID_GEN.next())
+
+@compiletime function createSpellElkCritter()
+    new AbilityDefinitionMechanicalCritter(SPELL_ELK_CRITTER_ID)

```

Comment:
No additional config for the spell? Not sure what the spell default object looks like but maybe add 0 cooldown, 0 manacost, 99999 range, no art etc

Reply:
@Marsunpaisti none of those things are configurable for this spell lol